test/e2e/scalability/load.go

```diff
@@ -703,3 +721,28 @@ func CreateNamespaces(f *framework.Framework, namespaceCount int, namePrefix str
 	}
 	return namespaces, nil
 }
+
+func CreateQuotas(f *framework.Framework, namespaces []*v1.Namespace, podCount int, testPhase *timer.Phase) error {
+	defer testPhase.End()
+	quotaTemplate := &v1.ResourceQuota{
+		Spec: v1.ResourceQuotaSpec{
+			Hard: v1.ResourceList{"pods": *resource.NewQuantity(int64(podCount), resource.DecimalSI)},
+		},
+	}
+
+	for _, ns := range namespaces {
+		if err := wait.PollImmediate(2*time.Second, 30*time.Second, func() (bool, error) {
+			quotaTemplate.Name = ns.Name + "-quota"
+			_, err := f.ClientSet.CoreV1().ResourceQuotas(ns.Name).Create(quotaTemplate)
+			if err != nil && !errors.IsAlreadyExists(err) {
+				framework.Logf("Unexpected error while creating resource quota: %v", err)
+				return false, nil
+			}
+			return true, nil
+		}); err != nil {
+			return err
```

Comment:
`fmt.Errorf("Failed to create quota: %v", err)`?
Otherwise sth less informative like "timed out waiting for condition" will show up.

Reply:
Done.